jerry-core/parser/js/js-lexer.c

```diff
@@ -366,6 +375,9 @@ static const keyword_string_t keyword_length_5[10] =
   LEXER_KEYWORD ("throw", LEXER_KEYW_THROW),
   LEXER_KEYWORD ("while", LEXER_KEYW_WHILE),
   LEXER_KEYWORD ("yield", LEXER_KEYW_YIELD),
+#ifndef CONFIG_DISABLE_ES2015
+  LEXER_KEYWORD ("await", LEXER_KEYW_AWAIT),
+#endif /* !CONFIG_DISABLE_ES2015 */
```

Comment:
Is this correct this way? The rest of the keywords seem to be alpha-sorted. Does the lexer not depend on the order?

Reply:
I don't think it is dependent on the alphabetic order, but let's follow the rules.